Scripts/script-LinkIncidentsWithRetry.yml

```diff
@@ -0,0 +1,29 @@
+commonfields:
+  id: LinkIncidentsWithRetry
+  version: -1
+name: LinkIncidentsWithRetry
+script: |
+  linkedIncidentIDs = demisto.args()["linkedIncidentIDs"]
+  counter = int(demisto.args()["retryLimit"])
+  res = demisto.executeCommand("linkIncidents", {"linkedIncidentIDs":linkedIncidentIDs})
+  while isError(res[0]) and counter>0 and ("DB Version" in res[0]["Contents"]):
+      time.sleep(2)
+      demisto.error(str(counter)+" retry linkIncidents:"+str(linkedIncidentIDs))
+      res = demisto.executeCommand("linkIncidents", {"linkedIncidentIDs":linkedIncidentIDs})
+      counter = counter - 1
+  demisto.results(res)
+type: python
+tags: []
+enabled: true
+args:
+- name: linkedIncidentIDs
+  required: true
+  description: The list of other incidents to link or mark as duplicate. Accepts comma-delimited
```

Comment:
What do you mean by `or mark as duplicated`?
And i only saw a test for adding single incident, lets try to make it for multiple incidents at the same time to make sure that it will actually work all the time, wouldn't you agree?

Reply:
add test for 2 incidents + removed the duplicate part